requests_test.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import json
+
+#getting start and end dates to get data from
+start = input("Enter start date in (YYYY/MM/DD) format: ")
+end = input("Enter end date in (YYYY/MM/DD) format: ")
+#using the esearch eutil to grab the 'Webenv' and 'total record' values to then send to efetch
+search_url = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&mindate="+start+"&maxdate="+end+"&usehistory=y&retmode=json"
+search_r = requests.post(search_url)

```

Comment:
Ask me for an EUTILS API key. We should figure out a way to include this as a URL parameter by declaring the API key as a command-line or environmental variable.

Reply:
Why use the HTTP POST method rather than GET?